### ISSUE TITLE
Accuracy is High but Validation Accuracy is Bad

### ISSUE THREAD

**Issue by xuhaoteoh** [KaroDievas/car-sound-classification-with-keras#9]
Hi, I have been working on the channel ordering issue and finally I managed to train the model but the results are very bad. Do you have any suggestions on how shall I improve the results?

The results are as attached below:

https://colab.research.google.com/drive/1xHghjvOEVZc3bGtLfgRNw6Xq51d7U_XA

https://colab.research.google.com/drive/1G_oh_f-2TxF8oa6bVpn-qXlRyOfAPAAq

**Comment by KaroDievas** [comment]
Hi, 

There are no clear answer to this question.
But there can be at least two approaches:
- increase number of sample data (cause it's a deep learning, that means more data you have better network can learn)
- experiment with the current model. You can add more layers, change activation functions.

Also it's highly expected that multi class is less accurate than simple binary.
